README.md

```diff
@@ -12,6 +12,7 @@
         * [Image with systemd as init](#image-with-systemd-as-init)
         * [Image with the agent as init](#image-with-the-agent-as-init)
     * [Initrd creation](#initrd-creation)
+* [Platform-Distro Compatibility Matrix](#Platform-Distro-Compatibility-Matrix)
```

Comment:
I think for the hotlink to work to the section you have to make all the words lowercase, like `#platform-distro-compatibility-matrix` - annoying though it is, I know...